install_config/install/prerequisites.adoc

```diff
@@ -361,6 +361,22 @@ $ dig node1.example.com @10.64.33.1 +short
 10.64.33.101
 ----
 
+[[dns-config-prereq-disabling-dnsmasq]]
+==== Disabling dnsmasq
+
+If you want to disable *dnsmasq* (for example, if your *_/etc/resolv.conf_* is
+managed by a configuration tool other than NetworkManager), then set
+`openshift_use_dnsmasq` to *false* in the Ansible playbook.
+
+However, certain containers do not properly move to the next nameserver when the
+first issues *SERVFAIL*. Red Hat Enterprise Linux (RHEL)-based containers do not
+suffer from this, but certain versions of *uclibc* and *musl* do.
+
+Enabling *dnsmasq* also enables pushing to the registry via the DNS name
```

Comment:
I do not understand this statement, are we talking about the route exposed for the registry or the SKYDNS service name. Even with out dnsmasq we should be able to use both if the cluster is configured correctly.  We inject the kubernetes service ip and search domains in the container resolv.conf  and take resolv.conf from the host. So we should be able to resolve everything still. 

Unless things have changed in later versions.